supabase_functions/_async/functions_client.py

```diff
@@ -30,15 +30,33 @@ def __init__(
             "User-Agent": f"supabase-py/functions-py v{__version__}",
             **headers,
         }
-        self._client = AsyncClient(
-            base_url=self.url,
-            headers=self.headers,
-            verify=bool(verify),
-            timeout=int(abs(timeout)),
-            proxy=proxy,
-            follow_redirects=True,
-            http2=True,
-        )
+
+        if timeout is not None:
```

Comment:
I like the idea of deprecating these and suggesting to handle this in your httpx client yourself.